README.md

```diff
@@ -1,29 +1,77 @@
-# likeminds-feed-iOS
+# Welcome to LikeMindsFeed SDK!
 
-[![CI Status](https://img.shields.io/travis/Devansh Mohata/likeminds-feed-iOS.svg?style=flat)](https://travis-ci.org/Devansh Mohata/likeminds-feed-iOS)
-[![Version](https://img.shields.io/cocoapods/v/likeminds-feed-iOS.svg?style=flat)](https://cocoapods.org/pods/likeminds-feed-iOS)
-[![License](https://img.shields.io/cocoapods/l/likeminds-feed-iOS.svg?style=flat)](https://cocoapods.org/pods/likeminds-feed-iOS)
-[![Platform](https://img.shields.io/cocoapods/p/likeminds-feed-iOS.svg?style=flat)](https://cocoapods.org/pods/likeminds-feed-iOS)
+LikeMindsFeed SDK offers an Instagram-like feed integration for your iOS applications, making it easy to add a social feed feature with minimal setup. The SDK is composed of two main components:
 
-## Example
-
-To run the example project, clone the repo, and run `pod install` from the Example directory first.
+- **LikeMindsFeedUI:** Contains all the reusable UI components.
+- **LikeMindsFeedCore:** Houses the actual implementation and business logic, and relies on `LikeMindsFeedUI` for UI components.
 
 ## Requirements
 
+- iOS 13.0 or later
+
 ## Installation
 
-likeminds-feed-iOS is available through [CocoaPods](https://cocoapods.org). To install
-it, simply add the following line to your Podfile:
+### Cocoapods
+
+You can easily integrate LikeMindsFeed into your project using Cocoapods. Add the following lines to your Podfile:
 
 ```ruby
-pod 'likeminds-feed-iOS'
+platform :ios, '13.0'
+use_frameworks!
+target 'MyApp' do
+    pod 'LikeMindsFeedCore'
+end
+```
+
+> **Note:** Support for Swift Package Manager (SPM) is on the way!
+
+## Integration
+
+To integrate `LikeMindsFeedCore` into your app, you need to implement two required functions:
+
+### 1. Setup LikeMindsFeed
+
+```swift
+func setupLikeMindsFeed(apiKey: String, analytics: LMFeedAnalyticsProtocol? = nil)
 ```
 
-## Author
+Use this function to set the API key for the SDK. It's recommended to call this as early as possible in your app's lifecycle.
 
-Devansh Mohata, devansh.mohata@likeminds.community
+**Example:** In your `AppDelegate.swift`:
 
-## License
+```swift
+func application(_ application: UIApplication, willFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]? = nil) -> Bool {
+    LMFeedCore.shared.setupLikeMindsFeed(apiKey: "Your_API_Key", analytics: nil)
+    return true
+}
+```
+
+### 2. Initiate LikeMindsFeed
+
+```swift
+func initiateLikeMindsFeed(username: String, userId: String, completionHandler: ((Result<Void, LMFeedError>) -> Void)?)
+```
+
+This function initiates the SDK. Initiating the feed is crucial as you won't be able to access the feed without this step.
```

Comment:
Add this like a note:
Initiating the feed is crucial as you won't be able to access the feed without this step.

Reply:
Done!